.github/workflows/ci.yaml

```
@@ -46,6 +50,12 @@ jobs:
       with:
         distribution: zulu
         java-version: ${{matrix.java}}
+    - name: Authenticate to Google Cloud
+      uses: google-github-actions/auth@67e9c72af6e0492df856527b474995862b7b6591 # v2.0.0
```

Since this is only needed for Flakybot then we can save time by only running auth when Flakybot is needed.
```suggestion
    - name: Authenticate to Google Cloud
      # only needed for Flakybot on periodic (schedule) and continuous (push) events
      if: ${{ github.event_name == 'schedule' || github.event_name == 'push' }}
      uses: google-github-actions/auth@67e9c72af6e0492df856527b474995862b7b6591 # v2.0.0
```

---

Updated. Thanks for the suggestion.